__tests__/database/migration/file/index.test.ts

```diff
@@ -7,6 +7,6 @@ describe('TESTING all migration file', () => {
     // Execution
 
     // Assertion
-    expect(allMigrations.length).toEqual(7);
+    expect(allMigrations.length).toEqual(8);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test is updated to expect 8 migrations instead of 7, which aligns with the addition of a new migration (`MigrationVersion7`) as mentioned in the PR summary. However, ensure that this new migration is included in the `allMigrations` array to avoid test failures.
```diff
-    expect(allMigrations.length).toEqual(7);
+    expect(allMigrations.length).toEqual(8);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->